metaflow/tracing.py

```diff
@@ -0,0 +1,28 @@
+import sys
+from metaflow.metaflow_config import (
+    OTEL_ENDPOINT,
+)
+
+import metaflow.tracing_noop
```

Comment:
I believe this could be cleaned up a bit with import aliases
```
import metaflow.tracing_noop as tracing
...
import metaflow.tracing_otel as tracing
...
init_tracing = tracing.init_tracing
cli_entrypoint = tracing.cli_entrypoint
...
```
it has the added benefit of acting similar to an interface, throwing errors if the latest imported `tracing` does not implement all the functions that are being assigned later on.

Reply:
Agree that it would be better to design an interface for all tracing endpoint. In the current design, we need to copy/paste line 16 to 26 to add another tracing plugin (e.g. ZIPKIN_ENDPOINT)